src/sentry/tasks/post_process.py

```diff
@@ -674,10 +674,7 @@ def process_snoozes(job: PostProcessJob) -> None:
         and group.status == GroupStatus.IGNORED
         and group.substatus == GroupSubStatus.UNTIL_ESCALATING
     ):
-        if not is_escalating(group):
-            job["has_reappeared"] = True
-            return
-        job["has_reappeared"] = False
+        job["has_reappeared"] = is_escalating(group)
```

Comment:
fix: we should've been setting `job["has_reappeared"] = True` if the job is escalating since it transitions to `escalating`, not the other way around